python/lsst/ts/lasertracker/laser_tracker_csc.py

```diff
@@ -650,6 +650,12 @@ async def measure_alignment(self, target: str) -> None:
             target=target_frame_name, reference_pointgroup=reference_frame_name
         )
 
+        # Remove Zeropoint from offset
+        if target == "M2":
+            target_offset -= self.config.m2_zeropoint
```

Comment:
if you use my suggestion on the configuration, this would be

```
target_offset -= np.array([self.config.zero_points["M2"][axis] for axis in "xyzuv"])
```